pkg/network/tracer/dump.go

```diff
@@ -0,0 +1,267 @@
+// +build linux_bpf
+
+package tracer
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/network/ebpf/probes"
+	"github.com/DataDog/ebpf/manager"
+
+	"github.com/davecgh/go-spew/spew"
+)
+
+func dumpMapsHandler(currentMap *manager.Map, manager *manager.Manager) string {
+	var output stings.Builder
+	switch currentMap.Name {
+
+	case "connectsock_ipv6": // maps/connectsock_ipv6 (BPF_MAP_TYPE_HASH), key C.__u64, value uintptr // C.void*
+		output.WriteString("eBPF Map: '" + currentMap.Name + "', key: 'C.__u64', value: 'uintptr // C.void*'")
+		iter := Map.Iterate()
```

Comment:
Note that `Iterate` doesn't work on older kernel versions. You have to use `IterateFrom` starting with a key that doesn't exist.

Reply:
minimum kernel versions is 4.4.132 from the godoc maps.Iterate()

Reply:
CentOS/RHEL have a kernel version earlier than that despite supporting some eBPF features because they were backported.